js/views/templates/roomlistview.handlebars

```diff
@@ -43,7 +43,9 @@
 				<span>{{copyLinkText}}</span>
 			</button>
 		</li>
-		<li><div class="separator"></div></li>
+		<li><div class="separator"></div>
+		<li class="app-navigation-entry-menu-caption">Chat notifications</li>
```

Comment:
Please use a placeholder similar to the other texts, so they can be translated:

```suggestion
		<li class="app-navigation-entry-menu-caption">{{notificationCaptionText}}</li>
```

And then add the label in the roomlistview.js as well:
```js
notificationCaptionText: t('spreed', 'Chat notifications'),
```

Reply:
Right, I'm on it!